docs/burial.rst

```diff
@@ -2,20 +2,23 @@ burial
 ======
 
 .. dfhack-tool::
-    :summary: Configures all unowned coffins to allow burial.
-    :tags: unavailable fort productivity buildings
+    :summary: Allows burial in unowned coffins.
+    :tags: fort productivity buildings
+
+Creates a 1x1 tomb zone for each built coffin that doesn't already have one.
```

Comment:
maybe "that isn't already in a zone" since it's possible for a zone to be larger than 1x1 and it's possible for a tomb zone to contain more than one coffin (perhaps done for roleplaying reasons)